core/ir.ml

```diff
@@ -39,58 +39,56 @@ type location = CommonTypes.Location.t
   [@@deriving show]
 
 type value =
-  [ `Constant of Constant.t
-  | `Variable of var
-  | `Extend of (value name_map * value option)
-  | `Project of (name * value)
-  | `Erase of (name_set * value)
-  | `Inject of (name * value * Types.datatype)
+  | Constant of Constant.t
```

Comment:
I'd align `of` in one column so that the declaration is visually more pleasant to read. Hint: in Emacs you can use `M-x align-regexp` on a block of text.